addons/web/static/src/views/list/list_arch_parser.js

```diff
@@ -153,6 +153,11 @@ export class ListArchParser extends XMLParser {
                 treeAttr.noOpen = archParseBoolean(node.getAttribute("no_open") || "");
                 treeAttr.expand = archParseBoolean(xmlDoc.getAttribute("expand") || "");
                 treeAttr.decorations = getDecoration(xmlDoc);
+
+                // custom open action when clicking on record row
+                const action = xmlDoc.getAttribute("action");
+                const type = xmlDoc.getAttribute("type") || "object";
```

Comment:
Not sure about the default value.

`<tree action="abc"/>` => `abc` is a method
`<tree action="abc" type="action"/>` => `abc` is an action

Doesn't look natural as the other attribute is named `action`.

Note: in Kanban, there's no default (same API as for `<button>`)

Reply:
Agreed and changed it to match the default behavior of the Kanban + Button behavior